api/query/cache/index/aggregator.go

```diff
@@ -72,10 +72,11 @@ func (a *indexAggregator) Done() []query.SearchResult {
 	return res
 }
 
-func newIndexAggregator(idx index, rus []RunID) aggregator {
+func newIndexAggregator(idx index, rus []RunID, opts query.AggregationOpts) aggregator {
 	return &indexAggregator{
```

Comment:
Syntax question - looks like &indexAggregator returns the reference to this object, which is a pointer. But how come the return type of the function is aggregator instead of *aggregator?